exonum/src/runtime/error.rs

```diff
@@ -88,7 +88,8 @@ use crate::{
 ///
 /// [`catch_unwind`]: https://doc.rust-lang.org/std/panic/fn.catch_unwind.html
 /// [`DispatcherError`]: ../enum.DispatcherError.html
-#[derive(Clone, Copy, Debug, Eq, Hash, Ord, PartialEq, PartialOrd)]
+#[derive(Clone, Copy, Debug, Deserialize, Serialize)]
```

Comment:
Implementing `(De)Serialize` is not necessary; see the comment about `ExecutionStatus`.